Add Social media handbook page (#17450)
PostHog/posthog.com: contents/handbook/marketing/social-media.md

---
title: Social media
sidebar: Handbook
showTitle: true
---

> **Got a launch, Beta, GA, or project you want promoted on social?** Tag <TeamMember name="Liam Graham" /> in Slack and use the checklist below to give him the information he needs.

> When we promote a launch on social, <TeamMember name="Liam Graham" /> needs a handful of things to turn it into great posts. Rather than asking everyone to fill out a separate form or doc, we keep this lightweight: just include a short checklist in the first or second message you share about the work.

Reviewer: ```suggestion
When we promote a launch on social, <TeamMember name="Liam Graham" /> needs a handful of things to turn it into great posts. 
```

As Andy would say "needlessly verbose"

> This page explains what to include and where to send it.

Reviewer: ```suggestion
```

> ## What to share

Reviewer: notably missing @liamg-posthog 's ask for the ELI5

In the initial (or second) message you share about a launch, Beta, GA, or project, add this bullet list where you'd normally tag <TeamMember name="Liam Graham" />:

- **Messaging doc(s).** Do you have email copy that lives somewhere? A general messaging doc? Send both if you have them.
- **Link(s) to the main video or photo asset(s).** As a gentle reminder, **4:5** aspect ratio is best for graphics and photos, and **9:16** is best for video.
- **Link(s) to the blog article**, if we've got one.
- **When should the first post on social go live?**
- **Confirm the OpenGraph image is updated on the product page.** See [OpenGraph images](#opengraph-images-for-product-pages) below.

That's the whole ask. If something doesn't apply (for example, there's no blog post yet), just say so.

## Where and when to send it

> You should aim to give as much notice as you can, but ideally at least a week. You can share the checklist in either of two places:

Reviewer: Does this need to be so variable? Can it just be one channel/process we always do?

Reviewer 2: Given there's only two options (I can see we've removed email), I think we're okay to have these two options. They're both the via same medium, and the same content.

- Post in either the #team-marketing or #team-editorial Slack channels, tagging <TeamMember name="Liam Graham" />.

We want to make sure we're being transparent with this, so you should avoid sending this information by DM.

## OpenGraph images for product pages

Every _product_ page should have a custom OpenGraph (OG) image, so that when someone shares a link on social media it shows a purpose-built visual rather than defaulting to the website homepage image or, worse, showing up as a missing image.

- **What's needed:** a custom OG image for each product page. This is an art request rather than a copy request, so there's usually no messaging doc to go with it.
- **Good examples:** the [Feature Flags](/feature-flags) and [Session Replay](/session-replay) product pages already have working OG images that render correctly. Use these as a reference for what we're aiming for.
- **Design guidance:** because these render on small screens, keep designs and copy simple – bright colors and chunky text work best.
- **Check your work:** paste the product page URL into [opengraph.xyz](https://www.opengraph.xyz/) to preview how the OG image renders across platforms.

> If you can sort the OG image yourself for a product as you pick up assignments, that's one less thing to coordinate. Tracking for this work lives in [this GitHub issue](https://github.com/PostHog/posthog.com/issues/16998).

Reviewer: Probably don't need to link a PR in flight? Will become stale.
In general I'm confused about the ownership of OG images as part of social media. Do we make an art request for the image and then give it to Liam? Or does Liam make the request based on the checklist we give him? Who owns updating the product page with it? How do we do that?

Reviewer 2: Throwing this in Slack

---
title: Social media
sidebar: Handbook
showTitle: true
---

> > **Got a launch, Beta, GA, or project you want promoted on social?** Include the checklist below in the message where you'd usually tag <TeamMember name="Liam Graham" />, and you're done. No extra form to fill out.

Author: ```suggestion
> **Got a launch, Beta, GA, or project you want promoted on social?** Tag <TeamMember name="Liam Graham" /> in Slack and use the checklist below to give him the information he needs.
```

When we promote a launch on social, <TeamMember name="Liam Graham" /> needs a handful of things to turn it into great posts. Rather than asking everyone to fill out a separate form or doc, we keep this lightweight: just include a short checklist in the first or second message you share about the work.

This page explains what to include and where to send it.

## What to share

In the initial (or second) message you share about a launch, Beta, GA, or project, add this bullet list where you'd normally tag <TeamMember name="Liam Graham" />:

> - **ELI5 of what's getting promoted, and how you want it presented on social.** Only needed if it's not already clear from the message.

Author: ```suggestion
```
- **ELI5 of what's getting promoted, and how you want it presented on social.** Only needed if it's not already clear from the message.
```

- **Messaging doc(s).** Do you have email copy that lives somewhere? A general messaging doc? Send both if you have them.
- **Link(s) to the main video or photo asset(s).** As a gentle reminder, **4:5** aspect ratio is best for graphics and photos, and **9:16** is best for video.
- **Link(s) to the blog article**, if we've got one.
- **When should the first post on social go live?**

> - **Confirm the OpenGraph image is updated on the product page.** See [OpenGraph images](#opengraph-images-for-product-pages) below.

Author: ```suggestion
- **Confirm the OpenGraph image is updated on the product page.** See [OpenGraph images](#opengraph-images-for-product-pages) below.
- 
```
```

That's the whole ask. If something doesn't apply (for example, there's no blog post yet), just say so.

> ## Where to send it

Author: ```suggestion
## Where and when to send it
```

> You can share the checklist in either of two places:

Author: ```suggestion
You should aim to give as much notice as you can, but ideally at least a week. You can share the checklist in either of two places:
```

- A widely broadcast Slack message you're already sending out about the launch.

> - A separate email to <TeamMember name="Liam Graham" />.

Author: ```suggestion
- Post in either the #team-marketing or #team-editorial Slack channels, tagging <TeamMember name="Liam Graham" />.
```

> If both are equally convenient for you, **email is Liam's preference**.

Author: ```suggestion
We want to make sure we're being transparent with this, so you should avoid sending this information by DM. 
```
